pyramid_ldap/__init__.py

```diff
@@ -129,13 +129,13 @@ def authenticate(self, login_unsafe, password_unsafe):
         :exc:`pyramid.exceptions.ConfiguratorError`.
 
         In pyramid_ldap <= version 0.2, authenticating with a login that
-        included the domain (e.g. CORP\exampleuser) would raise
-        ``ldap.FILTER_ERROR`` because the ``\`` led to an invalid search
+        included the domain (e.g. CORP\\exampleuser) would raise
```

Comment:
This should be a literal.
```suggestion
        included the domain (e.g., ``CORP\exampleuser``) would raise
```